assets/xml/objects/gameplay_keep.xml

```diff
@@ -1,77 +1,88 @@
-        <DList Name="gameplay_keep_DL_0027B0" Offset="0x27B0" />
-        <DList Name="gameplay_keep_DL_002838" Offset="0x2838" />
+        <DList Name="gKokiriSwordBladePart8DL" Offset="0x2480" />
+        <DList Name="gKokiriSwordBladePart7DL" Offset="0x2508" />
+        <DList Name="gKokiriSwordBladePart6DL" Offset="0x2590" />
+        <DList Name="gKokiriSwordBladePart5DL" Offset="0x2618" />
+        <DList Name="gKokiriSwordBladePart4DL" Offset="0x26A0" />
+        <DList Name="gKokiriSwordBladePart3DL" Offset="0x2728" />
+        <DList Name="gKokiriSwordBladePart2DL" Offset="0x27B0" />
+        <DList Name="gKokiriSwordBladePart1DL" Offset="0x2838" />
         <DList Name="gKokiriSwordBladeDL" Offset="0x28C0" />
-        <Texture Name="gameplay_keep_TLUT_002908" OutName="tlut_002908" Format="rgba16" Width="16" Height="16" Offset="0x2908" />
-        <Texture Name="gameplay_keep_Tex_002B08" OutName="tex_002B08" Format="rgba16" Width="8" Height="8" Offset="0x2B08" />
-        <Texture Name="gameplay_keep_Tex_002B88" OutName="tex_002B88" Format="rgba16" Width="16" Height="16" Offset="0x2B88" />
-        <Texture Name="gameplay_keep_Tex_002D88" OutName="tex_002D88" Format="ci8" Width="8" Height="8" Offset="0x2D88" />
-        <Texture Name="gameplay_keep_Tex_002DC8" OutName="tex_002DC8" Format="i8" Width="8" Height="8" Offset="0x2DC8" />
-        <Texture Name="gameplay_keep_Tex_002E08" OutName="tex_002E08" Format="rgba16" Width="16" Height="16" Offset="0x2E08" />
-        <Texture Name="gameplay_keep_Tex_003008" OutName="tex_003008" Format="i8" Width="8" Height="8" Offset="0x3008" />
+        <Texture Name="gKokiriSwordHandleTLUT" OutName="kokiri_sword_handle_tlut" Format="rgba16" Width="16" Height="16" Offset="0x2908" />
+        <Texture Name="gKokiriSwordHandlePommelTex" OutName="kokiri_sword_handle_pommel" Format="rgba16" Width="8" Height="8" Offset="0x2B08" />
+        <Texture Name="gRazorSwordHandleGripTex" OutName="razor_sword_handle_grip" Format="rgba16" Width="16" Height="16" Offset="0x2B88" />
+        <Texture Name="gKokiriSwordHandleCrossGuardTex" OutName="kokiri_sword_handle_cross_gaurd" Format="ci8" Width="8" Height="8" Offset="0x2D88" />
+        <Texture Name="gKokiriSwordHandleGripTex" OutName="kokiri_sword_handle_grip" Format="i8" Width="8" Height="8" Offset="0x2DC8" />
+        <Texture Name="gRazorSwordHandleDesignTex" OutName="razor_sword_handle_design" Format="rgba16" Width="16" Height="16" Offset="0x2E08" />
+        <Texture Name="gSwordMetalTex" OutName="sword_metal" Format="i8" Width="8" Height="8" Offset="0x3008" />
         <DList Name="gKokiriSwordDL" Offset="0x3050" />
         <DList Name="gRazorSwordDL" Offset="0x3060" />
+
         <DList Name="gDekuStickDL" Offset="0x32B0" /> <!-- @bug Supplies the wrong size to gsDPLoadTextureBlock, meaning it uses part of gLinkPauseChildJointTable as if it were a texture. -->
+
         <Blob Name="gameplay_keep_Blob_003400" Size="0x9D0" Offset="0x3400" />
```

Comment:
Looking at the binaries of this blob, I'm fairly sure this is a vtx of array count 157

Reply:
Any thoughts on the other one? I tried extracting as VTX, but it didn't like it. As an `RGBA16` it looks kind of like a TLUT, fits as an 16x16 one, and would make sense with the two other TLUTs nearby. However not sure that is enough evidence to make it one.

Reply:
All I can say is that's isn't definitely not Vtx from the binaries. `TLUT` does make the most sense but yeah hard to tell

Reply:
I visualized the Vtx here, and this is what I got. It's absolutely a mask of some kind, but I have zero clue what it could be:

https://github.com/zeldaret/mm/assets/12245827/2da82c5d-ff19-4465-b443-4fb7a8d93d74